draft-lassey-tigress-threat-model.md

```diff
@@ -116,13 +119,13 @@ From these goals we can derive a threat model for the general problem space.
 ## Assets and Data
 
 ### Credential
-The credential or key that is being transferred via this protocol.
+A digital credential {{Tigress-req-03}} is composed of Cryptographic material and other data that enables an user to access a property.
 
 ### Intermediary data
-Data that is transferred over the course of the transaction.
+Data that is exchanged over the course of credential transfer.
 
 ### Credential transfer invitation
-The initial data containing Provisioning Information {{Tigress-req-03}} transmetted to the receiver which represents an invitation to accept the transferred credential.
+The initial data containing Provisioning Information {{Tigress-req-03}} sent to the receiver. It represents an invitation to accept the transfer of copy of the credential.
```

Comment:
"to accept the transfer of the digital credential" ? Do we need to mention a "Copy"

Reply:
It's not necessary to mention copy. I'll remove it.